### PR TITLE
Add Actions#find_by

### PR DESCRIPTION
**Pull request by donaldong**
### Summar
This adds a find_by method which basically uses where. It'd be more performant if we can cast a limit on the where clause. See #45.

### Test Plan
Added a test case to cover the new behaviors.

### CODE REVIEW

**Comment by donaldong** [comment]
fix build

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redcord/compare/71e070f8ca6e26b87e0707702f052abefc0baaac..508e95a4e081a600d375ca8c505b98ce515e8203).

File: spec/actions_spec.rb [hunk 1]
```diff
@@ -137,6 +138,30 @@ def self.name
       expect(another_instance.id).to eq instance.id
       expect(another_instance.value).to eq instance.value
     end
+
+    it '#where' do
+      klass.establish_connection
```
Comment (katyho) [inline spec/actions_spec.rb:143]: Why do we need to call `establish_connection` for this test and the one below?

Comment (donaldong) [reply spec/actions_spec.rb:143]: we're resetting the redis database between examples. `establish_connection` would prepare the redis server (loading scripts and model definitions)

Comment (katyho) [reply spec/actions_spec.rb:143]: Ah right, and it fails here because model definitions with index attributes were flushed from redis. Makes sense!

**Pull request merged**